doc19org/medicam
# Send provider approval email

Steps to test:

1. Create a free/trial account with a SMTP service. (In this case I picked SendGrid).
2. Fill in the relevant environment variables.
3. Create a test provider in admin and set the email to your own. Set the status to approved and you should receive an email with the access URL.

Questions:
1. Wasn't exactly sure what the email message should be so let me know if it needs to be adjusted.
2. Do we need to worry about the case where an admin creates and approves a provider at the same time? Or will admins generally not be adding providers manually?
3. If the provider hasn't given an email can they still be approved? Or do we want to make the email field mandatory?
4. Should an email be sent if an admin revokes approval?
